### DISCUSSION TITLE
Contributing in other ways than development

### DISCUSSION THREAD

**Discussion by a3ng7n** [mbrlabs/Lorien#56] [Q&A] [answered]
I know it was mentioned in this post that money would never be charged; so I really like the idea of this app, and I want to see it succeed, but the only thing I have to give is money - is there a way we can donate or anything like that?

Also not sure if this is the place to ask this - feel free to nuke.

**Comment by mbrlabs** [comment]
Hey, thanks! I thought about setting up a Kofi page or something like that. When the app is more mature i also plan to setup an itch.io page where users can pay an optional amount of money if they want to support the development.

At the moment i'm busy with other projects though (don't worry, i'm still working on Lorien). If i set somthing up i'll ping you in here :)

**Comment by mbrlabs** [comment, marked as answer]
@a3ng7n so i setup a Kofi page: https://ko-fi.com/mbrlabs

You absolutly don't have to donate anything (to everyone who reads this!), but it's there if you want to buy me a coffee :blush:
